tests/de/gurkenlabs/litiengine/util/ArrayUtilitiesTests.java

```diff
@@ -95,4 +118,24 @@ public void testDistinct() {
 
     assertArrayEquals(new Integer[] { 1, 2, 3, 4, 5, 6 }, result);
   }
+
+  @Test
+  public void testContains() {
+    Object [] first = new Object[] { 1, 2, 3, 4, 5, null };
+    Object [] second = new Object[] {};
+    Object [] third = new Object[1];
+
+    assertEquals(true, ArrayUtilities.contains(first, 2));
```

Comment:
When checking for boolean conditions, use `assertTrue(...)` and `assertFalse(...)`. 
Please make sure this is done for all Tests that you introduced.